packages/features/schedules/components/DateOverrideList.tsx

```diff
@@ -39,62 +41,67 @@ const DateOverrideList = ({
   };
 
   return (
-    <ul className="rounded border border-gray-200" data-testid="date-overrides-list">
+    <List data-testid="date-overrides-list">
       {items.map((item, index) => (
-        <li key={item.id} className="flex justify-between border-b px-5 py-4 last:border-b-0">
-          <div>
-            <h3 className="text-sm text-gray-900">
-              {new Intl.DateTimeFormat("en-GB", {
-                weekday: "short",
-                month: "long",
-                day: "numeric",
-              }).format(item.ranges[0].start)}
-            </h3>
-            {item.ranges[0].end.getUTCHours() === 0 && item.ranges[0].end.getUTCMinutes() === 0 ? (
-              <p className="text-xs text-gray-500">{t("unavailable")}</p>
-            ) : (
-              item.ranges.map((range, i) => (
-                <p key={i} className="text-xs text-gray-500">
-                  {timeSpan(range)}
-                </p>
-              ))
-            )}
-          </div>
-          <div className="flex flex-row-reverse gap-5 space-x-2 rtl:space-x-reverse">
-            <DateOverrideInputDialog
-              excludedDates={excludedDates}
-              workingHours={workingHours}
-              value={item.ranges}
-              onChange={(ranges) => {
-                update(index, {
-                  ranges,
-                });
-              }}
-              Trigger={
-                <DialogTrigger asChild>
+        <>
+          <ListItem
+            removeHover
+            key={item.id}
+            heading={new Intl.DateTimeFormat("en-GB", {
```

Comment:
NIT heading in Figma is `font-normal` (instead of `font-semibold`)

Figma: 
<img width="557" alt="Screenshot 2023-03-02 at 10 46 35" src="https://user-images.githubusercontent.com/30310907/222477752-d667864c-53f6-4fd3-bfeb-24f25f0afe57.png">

Your changes:
<img width="558" alt="Screenshot 2023-03-02 at 10 52 36" src="https://user-images.githubusercontent.com/30310907/222480439-3250a97b-4326-4098-bf51-2a91f4238161.png">


Also button order changed, but actually looks better to me like that 🤔

Reply:
Hmm we may have this duplicated somewhere.

The new DS component has the correct styling for this 

https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=24489%3A159393